docs/docsite/rst/dev_guide/developing_modules_general.rst

```diff
@@ -64,6 +64,13 @@ Info and facts modules, are just like any other Ansible Module, with a few minor
 5. They MUST NOT make any changes to the system.
 6. They MUST document the :ref:`return fields<return_block>` and :ref:`examples<examples_block>`.
 
+You can add your ``*_facts`` into ``ansible_facts`` field of the result as follows:
```

Comment:
`*_facts` doesn't really make sense here, since the facts are not named `xxx_facts`. How about:
```suggestion
You can add your facts into ``ansible_facts`` field of the result as follows:
```